compiler/rustc_trait_selection/src/traits/const_evaluatable.rs

```diff
@@ -92,16 +93,7 @@ pub fn is_const_evaluatable<'tcx>(
             _ => bug!("unexpected constkind in `is_const_evalautable: {unexpanded_ct:?}`"),
         };
 
-        // FIXME: We should only try to evaluate a given constant here if it is fully concrete
-        // as we don't want to allow things like `[u8; std::mem::size_of::<*mut T>()]`.
```

Comment:
`try_evaluate_const` will be responsible for not evaluating if generic parameters are being used